src/app/categories/[categoryId]/pairwise-ranking/page.tsx

```diff
@@ -172,172 +179,112 @@ const CategoryPairwiseRankingPage = () => {
 						% of 100% Projects ranked
 					</p>
 				</div>
-				<p className='text-bold mb-4 mt-6 px-3 text-center text-base'>
+				<p className='text-bold mb-4 mt-6 px-3 text-center text-base font-bold'>
 					{`Which project should receive more RetroPGF funding in ${categoryData?.data.collection.name}?`}
 				</p>
 				<div className='items-top flex justify-between gap-4 pb-6 xxs:flex-col xs:flex-row'>
 					<div
 						key={firstProject.id}
-						onClick={() =>
-							!isLoading && handleVote(firstProject.id)
-						}
 						className={`${isLoading ? 'cursor-not-allowed opacity-50' : 'opacity-100'} cursor-pointer`}
 					>
 						<CategoryPairwiseCardWithMetrics
 							project={firstProject}
+							onClick={() =>
+								!isLoading && handleVote(firstProject.id)
+							}
+							onInfoClick={() => toggleDrawer(firstProject)}
 						/>
 					</div>
 					<div
 						key={secondProject.id}
-						onClick={() =>
-							!isLoading && handleVote(secondProject.id)
-						}
 						className={`${isLoading ? 'cursor-not-allowed opacity-50' : 'opacity-100'} cursor-pointer`}
 					>
 						<CategoryPairwiseCardWithMetrics
 							project={secondProject}
+							onClick={() =>
+								!isLoading && handleVote(secondProject.id)
+							}
+							onInfoClick={() => toggleDrawer(secondProject)}
 						/>
 					</div>
 				</div>
+
+				<Drawer setIsOpen={setIsDrawerOpen} isOpen={isDrawerOpen}>
+					<DrawerContent project={drawerProject} />
```

Comment:
I think there's no scenario in which `isDrawerOpen` is true but `drawerProject` is undefined, am I right?

If so, you can put a ! ([non-null assertion operator](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#non-null-assertion-operator)) in front of `drawerProject` and change the interface of `DrawerContent` component:

```ts
interface DrawerContentProps {
	project: IProject;
}
```